### ISSUE TITLE
Random sampling?

### ISSUE THREAD

**Issue by muschellij2** [skeydan/lesion_segmentation#5]
In the original training data, https://github.com/muschellij2/lesion_segmentation/blob/master/FLEXCONN_Train.py#L78 and https://github.com/muschellij2/lesion_segmentation/blob/master/FLEXCONN_Train.py#L85 do random sampling of the patches for training.  I find this a bit odd, I would assume you would do all the patches you have.  You can augment the data, but this seems hard for reproducibility.

I have added the `seed` option to most functions so that if this is `NULL`, all the patches are returned: https://github.com/muschellij2/flexconn/blob/master/R/mask_indices.R#L20, but otherwise it does random sampling: https://github.com/muschellij2/flexconn/blob/master/R/mask_indices.R#L29

This should help with the creation of volumes from patches as long as the indices are the same, which I wanted to bring up since you referenced that in https://github.com/muschellij2/flexconn/issues/1. 
@ccrainic

**Comment by skeydan** [comment]
I didn't think I referenced that topic? (in https://github.com/muschellij2/flexconn/issues/1)

But since you've brought it up - the way I understand it, this does only reorder the patches, not reduce them, right? Honestly to me  wasn't clear why they are doing it, given that keras shuffles the training data anyway, by default (you can change that).
But I just translated it and left it in, in case I was overlooking something...
Do you think I misunderstood this, and it does more than shuffling?

By the way, what do you think about my recent comments in https://github.com/muschellij2/flexconn/issues/1?

And regarding how we proceed in general, thanks for your pull request - but I was assuming that  we would switch to concentrating on the package? That's why I was asking how you see it regarding the package, 

- what should  be the entry point for the user, 
- what should the prediction functionality do / presuppose, 
- etc.
